app/mainwindow.ui

```diff
@@ -816,6 +817,17 @@
     <string>&amp;Diagram Support</string>
    </property>
   </action>
+  <action name="actionSourceAtOneSize">
```

Comment:
What do you think about adding this option to the "Editor" page in the options dialog instead? Does one really need fast access to this option or is this more a one time preference?